packages/schematics/angular/utility/latest-versions.ts

```diff
@@ -8,7 +8,7 @@
 
 export const latestVersions = {
   // These versions should be kept up to date with latest Angular peer dependencies.
-  Angular: '~8.0.0-beta.5',
+  Angular: '~8.0.0-beta.6',
   RxJs: '~6.4.0',
   ZoneJs: '~0.8.26',
   TypeScript: '~3.2.2',
```

Comment:
This should also be updated. As we should scaffold projects using TypeScript 3.3